src/ert/storage/local_ensemble.py

```diff
@@ -498,17 +434,96 @@ def get_ensemble_state(self) -> List[RealizationStorageState]:
             List of realization states.
         """
 
-        def _find_state(realization: int) -> RealizationStorageState:
+        def _parameters_exist_for_realization(realization: int) -> bool:
```

Comment:
Can you move this back so the diff is easier to read?

Reply:
Should show diff now, but I think there is no diff in functionality for these
https://github.com/equinor/ert/pull/9047/commits/3772cc776fedcab9e172039443edf9cb22879740

Reply:
Not sure I understand? Looks like these functions were just moved around? If you put it in a separate commit like now I guess it is fine, but did not see the reason for doing so?

Reply:
Only used from within that, so no need for it to be available outside that scope, unless we need it to be?